### PR TITLE
Link with rdynamic and -ldl

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

**Comment by davidbeckingsale** [comment]
LGTM

**Comment by mcfadden8** [comment]
LGTM

**Comment by mcfadden8** [comment]
LGTM

**Review by mcfadden8 — APPROVED** [review]
Love it!

File: CMakeLists.txt [hunk 1]
```diff
@@ -59,7 +59,7 @@ set(BLT_CXX_STD "c++11" CACHE STRING "Version of C++ standard")
 if (ENABLE_CUDA)
   cmake_minimum_required(VERSION 3.9)
 else ()
-  cmake_minimum_required(VERSION 3.8.2)
+  cmake_minimum_required(VERSION 3.8)
```
Comment (davidbeckingsale) [inline CMakeLists.txt:62]: This is a mistake, I'm going to fix it prior to merge.

Comment (mcfadden8) [reply CMakeLists.txt]: lol - I wondered about that but thought maybe it was intentional

**Comment by davidbeckingsale** [comment]
LGTM

**Review by mcfadden8 — APPROVED** [review]

**Pull request merged**